conversations/v3/whatsapp.yaml

```diff
@@ -2487,7 +2485,7 @@ components:
             c9468969-a3a3-4c17-95f7-1e8b2902ed74
         facebookBusinessManagerId:
           description: The Facebook Business Manager ID provided by Facebook
-          type: string
+          type: number
```

Comment:
Both examples and the implementation returns a number. Is that right?

Reply:
correct